src/locales/hr-HR.json

```
@@ -3,7 +3,8 @@
     "en-US": "Engleski",
     "fr-FR": "Francuski",
     "hr-HR": "Hrvatski",
-    "de-DE": "Njemački"
+    "de-DE": "Njemački",
+    "pl-PL": "Polirati"
```

This means "to polish" :D , should be "Poljski"

```suggestion
    "pl-PL": "Poljski"
```

---

Sorry :) I used Google to help with this. Thank you for correction 👍